<HTML>
<HEAD>
<TITLE>Mouse Genome Informatics: Other Contributed Data Sets and Tools</TITLE>

<!-- 
Author:   R. Palazola
Date:     3/23/1999
Project:  TR:121 -- Electronic Data Sets

NOTES:
- This will be our generic method for accessing MGI FTP data from the MGI Web Site.

- The top of the page contains info generic to all FTP/electronic data.

- Each available data set can/should have a Journal reference.  Originally
  Janan asked these be hyper-links to the Reference Detail page at our
  public MGI Web Site; due to concerns about maintaining this
  information and its links to MGI, Janan agreed to remove the
  hyper-link as the primary benefit to the user is the reference
  citation. I have left the html code in (commented out) for the
  original links in case this compromise is reversed.

- edited by jsb on 2/29/00 to add the strain genealogy data set
- edited by lec on 11/30/01 to add the Prows data set
- edited by lec on 01/19/2004 to add NIA Gene Index (TR 5471)
- edited by lec on 09/11/2012 to add Dr. David S. Park (TR11162)

08/14/2008	lec
	TR 9029; added FANTOM table

THE SOURCE FOR THIS FILE IS CONTROLLED BY RCS ON THE CORRESPONDING 
DEVELOPMENT FTP DIRECTORY (rohan:/export/rohan/ftp/pub/datasets).
PLEASE MAKE ALL MODIFICATIONS TO THE RCS SOURCE AND COPY TO PUBLIC.

-->

</HEAD>
<BODY BGCOLOR="#FFFFFF">

<H2>Mouse Genome Informatics: Other Contributed Data Sets and Tools</H2>
<P>
These data sets have been contributed to MGI by the authors. 
Currently they are not directly incorporated into the relational 
database. They are provided here as text or Excel files for the 
scientific community to view or download. 
 
We encourage contributions of such specialized important data 
sets that may be outside the structured information maintained
in MGI as a vehicle for community collaborations.
 
For data submission guidelines, see 
<A HREF="http://www.informatics.jax.org/submit.shtml">http://www.informatics.jax.org/submit.shtml</A>
or send email to <A HREF="http://www.informatics.jax.org/mgihome/support/mgi_inbox.shtml">User Support</A>.
<P>

<B>NOTE:</B> &quot;<I>Viewable Text</I>&quot; formatted pages are best viewed
with your browser set to use a fixed-width font such as Courier.
<P>
<HR>
	<B>Table of Contents</B>
	<OL>
	<LI><A HREF="#enu">ENU Incidental mutation data files</A>
	<LI><A HREF="#dotsdfci">MGI-DoTS and MGI-DFCI Mouse Gene Index Associations</A>
        <LI><A HREF="#NIA">MGI NIA Mouse Gene Index Associations</A>
        <LI><A HREF="#rhmap">The T31 RH Mapping Panel for the Laboratory Mouse and the Associated MGI Markers</A>
        <LI><A HREF="#major_histo"> Major Histocompatibility Complex H2 Haplotypes</A>
        <LI><A HREF="#genealogy">The Genealogy Chart of Inbred Strains</A>
        <LI><A HREF="#Moore">Polymorphisms in Microsatellite Markers for 129X1/SvJ vs Other Inbred Strains</A>
        <LI><A HREF="#macs">FileMaker Templates for Marker Assisted Congenic Screening (MACS) Database</A>
        <LI><A HREF="#MGIRI">RI Strain Distribution Patterns (MGI)</A>
        <LI><A HREF="#Paigen">RI Strain Distribution Patterns for AXB and BXA (Paigen)</A>
        <LI><A HREF="#Prows">RI Strain Distribution Patterns for AXB and BXA (Prows)</A>
        <LI><A HREF="#sslp">Inferring EST Map Positions Using SSLP Markers</A>
        <LI><A HREF="#elliott">Mouse Genome Sequence Based Maps of Nineteen Autosomes and Chromosome X</A>
        <LI><A HREF="#2cellembyro">2-Cell Embryo Gene List</A>
        <LI><A HREF="#mit">MIT STS Marker Data Files</A>
	<LI><A HREF="#ko">ES cell in situ hybridization data (M.S. Ko)</A>
	<LI><A HREF="#fantom">FANTOM Consortium Gene Ontology Annotations</A>
	<LI><A HREF="#DavidSPark">Exome Sequencing data used to identify candidate genes modifying the Parkinson's phenotype of the Park7 gene</A>
	</OL>
	<HR>

<OL>

    <LI><A NAME="enu"><B>ENU Incidental mutation data files</B></A><BR>
	   Data sets are provided by ENU mutagenesis centers that list putative mutations
	   discovered by sequencing genomes of mutagenized mice.  These may be
	   present in stocks generated by these projects.  Files are available as MS-Excel
	   spreadsheets.
	   <P>

           <UL>
           <LI><A HREF="incidental_muts/Mutagenetix.xlsx">Mutagenetix.xlsx</A>
           <LI><A HREF="incidental_muts/APF.xlsx">APF.xlsx</A>
	   </UL>

           <HR>

    <LI><A NAME="dotsdfci"><B>MGI-DoTS and MGI-DFCI Mouse Gene Index Associations</B></A><BR>
	    Reference:
	    J:79300, Zhu Y, Integrating Computationally Assembled Mouse Transcript Sequences (DoTS) 
	    with the Mouse Genome Informatics (MGI) Database.<BR>
	    <BR>
	    Reference:
	    J:79330, Zhu Y, Integrating Computationally Assembled Mouse Transcript Sequences (DFCI Mouse Gene Index) 
	    with the Mouse Genome Informatics (MGI) Database.<BR>
	    <BR>
           Tab-Delimited text/data.<BR>
	   Fields:  MGI ID, DT/TC ID, list of nucleotide sequence IDs which support the association.
	   <P>

           <UL>
           <LI><A HREF="DOTS/MGI_DT_via_GB_one_one">MGI-DT one-to-one</A>
           <LI><A HREF="DOTS/MGI_DT_via_GB_one_many">MGI-DT one-to-many</A>
           <LI><A HREF="DOTS/MGI_DT_via_GB_many_one">MGI-DT many-to-one</A>
           <LI><A HREF="DOTS/MGI_DT_via_GB_many_many">MGI-DT many-to-many</A>
           <LI><A HREF="DFCI/MGI_TC_via_GB_one_one">MGI-TC one-to-one</A>
           <LI><A HREF="DFCI/MGI_TC_via_GB_one_many">MGI-TC one-to-many</A>
           <LI><A HREF="DFCI/MGI_TC_via_GB_many_one">MGI-TC many-to-one</A>
           <LI><A HREF="DFCI/MGI_TC_via_GB_many_many">MGI-TC many-to-many</A>
           </UL>
           <HR>

    <LI><A NAME="NIA"><B>MGI NIA Mouse Gene Index Associations</B></A><BR>
	    Reference:
	    J:87300, MGI Sequence Group, Integrating Computationally Assembled Mouse Transcript Sequences (NIA Mouse Gene Index) 
	    with the Mouse Genome Informatics (MGI) Database.<BR>
	    <BR>
           Tab-Delimited text/data.<BR>
	   Fields:  MGI ID, NIA Mouse Gene Index transcript ID, list of sequence IDs 
	   (RefSeq, ENSEMBL transcripts and GenBank sequence accession ID) which support the association.
	   <P>

           <UL>
           <LI><A HREF="NIA/MGI_NIA_via_GB_one_one">MGI-NIA one-to-one</A>
           <LI><A HREF="NIA/MGI_NIA_via_GB_one_many">MGI-NIA one-to-many</A>
           <LI><A HREF="NIA/MGI_NIA_via_GB_many_one">MGI-NIA many-to-one</A>
           <LI><A HREF="NIA/MGI_NIA_via_GB_many_many">MGI-NIA many-to-many</A>
           </UL>
	   <HR>

  <LI><A NAME="rhmap"><B>The T31 RH Mapping Panel for the Laboratory Mouse and the Associated MGI Markers</B></A><BR>
	    Reference:
	    J:68900, The Jackson Laboratory Mouse Radiation Hybrid Database, Mouse T31 Radiation Hybrid Data Load.
	    <BR>
	   <P>

           <UL>
           <LI><A HREF="rhmap/README.txt">README.txt</A>
           <LI><A HREF="rhmap/RHpanelMGIReport.txt">RHpanelMGIReport.txt (Viewable Text)</A>
           <LI><A HREF="rhmap/RHpanelMGIReport.xls">RHpanelMGIReport.xls (MS-Excel Spreadsheet)</A>
           </UL>
	   <HR>

  <LI><A NAME="major_histo"><B> Major Histocompatibility Complex H2 Haplotypes</B></A><BR>
           Reference: 
           <A HREF="http://jaxmice.jax.org/literature/catalog/mhc_h2_haplotypes.pdf">Appendix III of JAX Mice Catalog</A>, 
	     HTML version by MGI Staff.
	     <P>
           <UL>
           <LI><A HREF="misc/H2Haplotypes/H2_haplotypes.html">Hypertext (.html)</A>
           </UL>
           <HR>

  <LI><A NAME="genealogy"><B>The Genealogy Chart of Inbred Strains</B></A><BR>
	    Reference:
	    J:59229, Beck JA, Nat Genet 2000;24():23-25.<P>
	    <UL>
	    <LI><A HREF="misc/genealogy/genealogy.pdf">Chart (.pdf)</A>
	    <LI><A HREF="misc/genealogy/genealogy_key.html">Key to Chart (.html)</A>
	    </UL>
	    <HR>

  <LI><A NAME="Moore"><B>Polymorphisms in Microsatellite Markers for 129X1/SvJ vs Other Inbred Strains</B></A><BR>
	   Reference: 
	   J:43843, Markel P; Shu P; Ebeling C; Carlson GA; Nagle DL;
	   Smutko JS; Moore KJ, Theoretical and empirical issues for
	   marker-assisted breeding of congenic mouse strains.,
	   Nat Genet 1997 Nov; 17(3):280-4
	   <P>
	   <UL>
	   <LI><A HREF="misc/Moore/Moore.txt"> Viewable Text (.txt)</A>
	   <LI><A HREF="misc/Moore/Moore.tsv"> Tab-Delimited text/data (.tsv)</A>
	   <LI><A HREF="misc/Moore/Moore.xls"> MS-Excel Spreadsheet</A>
	   </UL>
  	   <HR>

  <LI><A NAME="macs"><b>FileMaker Templates for Marker Assisted Congenic Screening (MACS) Database</B></A><BR>
       MACS stores three classes of information (husbandry, genotype analysis and phenotype screening) that are central to 
       the development of speed congenic and consomic lines. Downloadable files include a Help document of explanation and 
       a series of FileMaker 5.0 templates. The FileMaker files have been tested on MAC OS (7.6.1 to 9.2) and 
       Windows (95, 98, NT 4.0).
       <BR><BR>
       Reference: 
       J:82041, Collins, SC; Wallis, RH; Wallace K; McAteer M; Bihoreau MT; Gauguier D, 
       Marker Assisted Congenic Screening (MACS): A Database Tool for the Efficient Production and Characterization of 
       Congenic Lines., Mamm Genome 2003 (in press)
       <P>
       <UL>
       <LI><A HREF="MACS/help.doc">Help document (.doc)</A>
       <LI><A HREF="MACS/ANIMAL.FM">Animal.fm</A>
       <LI><A HREF="MACS/BREEDERS.FM">Breeders.fm</A>
       <LI><A HREF="MACS/CAGES.FM">Cages.fm</A>
       <LI><A HREF="MACS/LITTERS.FM">Litters.fm</A>
       <LI><A HREF="MACS/PEDIGREE.FM">Pedigree.fm</A>
       <LI><A HREF="MACS/PHENOT1.FM">Phenotype.fm</A>
       <LI><A HREF="MACS/PLATEP1.FM">Plate.fm</A>
       <LI><A HREF="MACS/GENEAL1.FM">Geneal.fm</A>
       <LI><A HREF="MACS/MARKER1.FM">Marker.fm</A>
       <LI><A HREF="MACS/JOINFI1.FM">JoinFI1.fm</A>
       <LI><A HREF="MACS/JOINFI2.FM">JoinFI2.fm</A>
       </UL>
       <HR>

  <LI><A NAME="MGIRI"><B>RI Strain Distribution Patterns</B></A><BR>
	   <P>
	   <UL>
           <LI><A HREF="RI/MGIRI/MGI_RISet_129XB.rpt">129XB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_58NXL.rpt">58NXL data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_9XA.rpt">9XA data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_AKXD.rpt">AKXD data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_AKXL.rpt">AKXL data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_AXB.rpt">AXD data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_B6NXC3N.rpt">B6NXC3N data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_BRX58N.rpt">BRX58N data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_BXA.rpt">BXA data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_BXD.rpt">BXD data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_BXH.rpt">BXH data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_BXJ.rpt">BXJ data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CBXE.rpt">CBXE data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CcS.rpt">CcS data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CX8.rpt">CX8 data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CXB.rpt">CXB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CXD.rpt">CXD data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CXJ.rpt">CXJ data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_CXS.rpt">CXS data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_EXCB.rpt">EXCB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_HcB(N3).rpt">HcB(N3) data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_HcB(N4).rpt">HcB(N4) data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_HPEiTy.rpt">HP/EiTy data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_LSXSS.rpt">LDXSS data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_LTRe.rpt">LT/Re data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_LTXB.rpt">LTXB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_LXB.rpt">LXB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_LXPL.rpt">LXPL data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_NOcCB.rpt">NOcCB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_NX129.rpt">NX129 data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_NX8.rpt">NX8 data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_NXL.rpt">NXL data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_NXSM.rpt">NXSM data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_OcB.rpt">OcB data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_OXA.rpt">OXA data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_SEAGnJ.rpt">SEA/GnJ data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_SMXA.rpt">SMXA data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_SWXJ.rpt">SWXJ data set</A>
           <LI><A HREF="RI/MGIRI/MGI_RISet_SWXL.rpt">SWXL data set</A>
	   </UL>
	   <HR>
	
  <LI><A NAME="Paigen"><B>RI Strain Distribution Patterns for AXB and BXA</B></A><BR>
	   Specially curated data sets available as <I>MS-Excel Spreadsheets</I>, by Chromosome.<BR><br>
	   Reference: 
	   J:45463, Sampson ST; Higgins DC; Elliot RW; Taylor BA; Lueders KK;
	   Koza RA; Paigen B, An edited linkage map for the AXB and BXA
	   recombinant inbred mouse., Mamm Genome 1998 SEP; 9(9):688-694
	   <P>
	   <UL>
	   <LI><A HREF="RI/Paigen/AXB,BXA.QTL/">QTL data sets</A>
	   <LI><A HREF="RI/Paigen/AXB,BXA.complete/">Complete data sets</A>
	   </UL>
	   <HR>
	
  <LI><A NAME="Prows"><B>RI Strain Distribution Patterns for AXB and BXA</B></A><BR>
           Reference:
	   Prows, D.R. and Horner, M.L.  Parental genetic contributions in the AXB and BXA recombinant
	   inbred mouse strains. Mammalian Genome 13: 127-133 (2002)
	   <P>
           <UL>
	   <LI><A HREF="misc/Prows/Prows-AXB.xls">AXB - MS-Excel Spreadsheet</A>
	   <LI><A HREF="misc/Prows/Prows-BXA.xls">BXA - MS-Excel Spreadsheet</A>
           </UL>
           <HR>

  <LI><A NAME="sslp"><B>Inferring EST Map Positions Using SSLP Markers</B></A><BR>
	   Reference: 
	   J:49040, Herron BJ; Silva GH; Flaherty L, Putative assignments
	   of ESTs to the genetic map by use of the SSLP database,
	   Mamm Genome 1998 Dec;9(12):1072-4.
	   <P>
	   <UL>
	   <LI><A HREF="misc/Flaherty/herron.txt">Viewable Text (.txt)</A>
	   <LI><A HREF="misc/Flaherty/herron.tsv">Tab-Delimited text/data (.tsv)</A>
	   <LI><A HREF="misc/Flaherty/herron.xls">MS-Excel Spreadsheet</A>
	   </UL>
	   <HR>

  <LI><A NAME="elliott"><B>Mouse Genome Sequence Based Maps of Nineteen Autosomes and Chromosome X</B></A><BR>
      Reference:
          Dr. Rosemary Elliott, Roswell Park Cancer Institute, Buffalo, New York. rosemary.elliott@roswellpark.org
      <P>
      <UL>
          <LI><A HREF="misc/Elliott/Methods_and_Caveats.doc">Methods and Caveats (.doc)</A></LI>
          <LI><A HREF="misc/Elliott/Chr_1_2005S.xls">Chromosome 1 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_2_2005S.xls">Chromosome 2 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_3_2005S.xls">Chromosome 3 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_4_2007.xls">Chromosome 4 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr5_2005S.xls">Chromosome 5 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_6_2003_S.xls">Chromosome 6 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_7_2005S.xls">Chromosome 7 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_8_2004_S.xls">Chromosome 8 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr9_2005S.xls">Chromosome 9 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_10_2004_S.xls">Chromosome 10 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_11_2004S.xls">Chromosome 11 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_12_2004_S.xls">Chromosome 12 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_13_2004.xls">Chromosome 13 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_14_2004S.xls">Chromosome 14 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_15_2007.xls">Chromosome 15 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_16_2004S.xls">Chromosome 16 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_17_2003_S.xls">Chromosome 17 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_18_2007.xls">Chromosome 18 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_19_2002_openS.xls">Chromosome 19 - MS-Excel Spreadsheet</A></LI>
          <LI><A HREF="misc/Elliott/Chr_X_2002_seqS.xls">Chromosome  X - MS-Excel Spreadsheet</A></LI>
      </UL>
      <HR>

  <LI><A NAME="2cellembyro"><B>2-Cell Embryo Gene List</B></A><BR>
      Reference:
          Evsikov AV, de Vries WN, Peaston AE, Radford EE, Fancher KS, Chen FH, Blake JA, Bult CJ, Latham KE, Solter D, Knowles BB. 2004. Systems biology of the 2-cell mouse embryo. Cytogenet Genome Res. 105(2-4):240-50. 
	  <P>
      <UL>
          <LI><A HREF="misc/Bult/2cellembryogenes_ver2_1.xls">MS-Excel Spreadsheet</A></LI>
      </UL>
      <HR>

  <LI><A NAME="mit"><B>MIT STS Marker Data Files With Release Data</B></A><BR>
      Reference:
	  J:34136, Whitehead Institute/MIT, Center for Genome Research, Database Release 1999;()
	   <P>
	   <UL>
	   <LI><A HREF="misc/MIT/MIT_05-99.MarkerInfo.txt">MIT Original Marker Name, Assay Name, Primers, etc. - Viewable Text (.txt)</A>
	   <LI><A HREF="misc/MIT/MIT_05-99.Sequences.txt">Full Assay Sequence - Viewable Text (.txt)</A>
	   <LI><A HREF="misc/MIT/oct93/Alternate_Assays.txt">Oct 93 - Alternate Assays.txt</A>
	   <LI><A HREF="misc/MIT/oct93/Map_Statistics.txt">Oct 93 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/oct93/Marker_Data.txt">Oct 93 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/oct93/Sequences.txt">Oct 93 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/jan94/Further_Assays.txt">Jan 94 - Further Assays.txt</A>
	   <LI><A HREF="misc/MIT/jan94/Map_Statistics.txt">Jan 94 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/jan94/Marker_Data.txt">Jan 94 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/jan94/Sequences.txt">Jan 94 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/jul94/Alternate_Assays.txt">Jul 94 - Alternate Assays.txt</A>
	   <LI><A HREF="misc/MIT/jul94/Map_Statistics.txt">Jul 94 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/jul94/Marker_Data.txt">Jul 94 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/jul94/Sequences.txt">Jul 94 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/oct94/Alternate_Assays.txt">Oct 94 - Alternate Assays.txt</A>
	   <LI><A HREF="misc/MIT/oct94/Map_Statistics.txt">Oct 94 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/oct94/Marker_Data.txt">Oct 94 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/oct94/Sequences.txt">Oct 94 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/jan95/Alternate_Assays.txt">Jan 95 - Alternate Assays.txt</A>
	   <LI><A HREF="misc/MIT/jan95/Map_Statistics.txt">Jan 95 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/jan95/Marker_Data.txt">Jan 95 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/jan95/Sequences.txt">Jan 95 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/apr95/Alternate_Assays.txt">Apr 95 - Alternate Assays.txt</A>
	   <LI><A HREF="misc/MIT/apr95/Map_Statistics.txt">Apr 95 - Map Statistics.txt</A>
	   <LI><A HREF="misc/MIT/apr95/Marker_Data.txt">Apr 95 - Marker Data.txt</A>
	   <LI><A HREF="misc/MIT/apr95/Sequences.txt">Apr 95 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/jun96/06-96.Contig2STS.txt">Jun 96 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/jun96/06-96.GeneticMapInfo.txt">Jun 96 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/jun96/06-96.MarkerInfo.txt">Jun 96 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/jun96/06-96.PhysicalMapInfo.txt">Jun 96 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/jun96/06-96.STS2Contig.txt">Jun 96 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/jun96/Sequences.txt">Jun 96 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/oct96/10-96.Contig2STS.txt">Oct 96 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/oct96/10-96.GeneticMapInfo.txt">Oct 96 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct96/10-96.MarkerInfo.txt">Oct 96 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct96/10-96.PhysicalMapInfo.txt">Oct 96 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct96/10-96.STS2Contig.txt">Oct 96 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/oct96/Sequences.txt.gz">Oct 96 - Sequences.txt.gz</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.Contig2STS.txt">Mar 97 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.DoubleContig2STS.txt">Mar 97 - DoubleContig2STS.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.GeneticMapInfo.txt">Mar 97 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.MarkerInfo.txt">Mar 97 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.PhysicalMapInfo.txt">Mar 97 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.STS2Contig.txt">Mar 97 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.STS2DoubleContig.txt">Mar 97 - STS2DoubleContig.txt</A>
	   <LI><A HREF="misc/MIT/mar97/3-97.Sequences.txt.gz">Mar 97 - Sequences.txt.gz</A>
	   <LI><A HREF="misc/MIT/may97/5-97.Contig2STS.txt">May 97 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.DoubleContig2STS.txt">May 97 - DoubleContig2STS.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.GeneticMapInfo.txt">May 97 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.MarkerInfo.txt">May 97 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.PhysicalMapInfo.txt">May 97 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.STS2Contig.txt">May 97 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.STS2DoubleContig.txt">May 97 - STS2DoubleContig.txt</A>
	   <LI><A HREF="misc/MIT/may97/5-97.Sequences.txt">May 97 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.Contig2STS.txt">Oct 97 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.DoubleContig2STS.txt">Oct 97 - DoubleContig2STS.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.GeneticMapInfo.txt">Oct 97 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.MarkerInfo.txt">Oct 97 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.PhysicalMapInfo.txt">Oct 97 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.STS2Contig.txt">Oct 97 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.STS2DoubleContig.txt">Oct 97 - STS2DoubleContig.txt</A>
	   <LI><A HREF="misc/MIT/oct97/10-97.Sequences.txt">Oct 97 - Sequences.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.Contig2STS.txt">May 99 - Contig2STS.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.DoubleContig2STS.txt">May 99 - DoubleContig2STS.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.GeneticMapInfo.txt">May 99 - GeneticMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.MarkerInfo.txt">May 99 - MarkerInfo.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.PhysicalMapInfo.txt">May 99 - PhysicalMapInfo.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.STS2Contig.txt">May 99 - STS2Contig.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.STS2DoubleContig.txt">May 99 - STS2DoubleContig.txt</A>
	   <LI><A HREF="misc/MIT/may99/05-99.Sequences.txt">May 99 - Sequences.txt</A>
	   </UL>
  	   <HR>

  <LI><A NAME="ko"><B>ES cell in situ hybridization data</B></A><BR>
	   Reference: 
	   J:129777, Carter MG; Stagg CA; Falco G; Yoshikawa T; Bassey UC; Aiba 
	   K; Sharova LV; Shaik N; Ko MS, An in situ hybridization-based screen for 
	   heterogeneously expressed genes in mouse ES cells., Gene Expr Patterns 2008 
	   8(3):181-98.
	   <P>
	   <UL>
	   <LI><A HREF="misc/Ko/table1.html">Table 1 (html)</A>
	   </UL>
	   <HR>

  <LI><A NAME="fantom"><B>FANTOM Consortium Gene Ontology AnnotationS</B></A><BR>
           The file below contains Gene Ontology (GO) annotations contributed by the 
	   FANTOM Consortium and RIKEN Genome Exploration Research Group, as described in the 
	   following publications:
	   <P>
           Okazaki Y, et al. 2002. Analysis of the mouse transcriptome based on functional annotation 
	   of 60,770 full-length cDNAs. Nature. 2002 Dec 5;420(6915):563-73.  
	   <P>
	   Carninci P, et al. 2006.  The transcriptional landscape of the mammalian genome. Science. 2005 Sep 2;309(5740):1559-63.
	   <P>
           The file is in standard GO gene association file format 
	   (see: <A HREF="http://www.geneontology.org/GO.format.annotation.shtml">http://www.geneontology.org/GO.format.annotation.shtml</A>
	   for details).
	   <P>
	   <UL>
	   <LI><A HREF="misc/FANTOM/gene_association_RCAfantom.mgi">gene_association_RCAfantom.mgi</A>
	   </UL>
	   <HR>

  <LI><A NAME="DavidSPark"><B>Exome Sequencing data used to identify candidate genes modifying the Parkinson's phenotype of the Park7 gene</B></A><BR>
	   Reference: J:186833, Rousseaux, MWC et al Progressive dopaminergic cell loss with unilateral-to-bilateral progression in a genetic model of Parkinson's disease, PNAS Sept, 2012
	   <P>
	   <UL>
	   <LI><A HREF="misc/DavidSPark/DJ1-C57-EXOMEanotated.xlsx">DJ1-C57-EXOMEanotated.xlsx</A>
	   </UL>
	   <HR>

</OL>

<P>
</BODY>
</HTML>
